xml/templates/message_templates/membership_online_receipt_html.tpl

```diff
@@ -190,28 +178,18 @@
       </tr>
     {/if}
 
-    {if !empty($trxn_id)}
+    {if {contribution.trxn_id|boolean}}
       <tr>
        <td {$labelStyle}>
         {ts}Transaction #{/ts}
        </td>
        <td {$valueStyle}>
```

Comment:
@eileenmcnaughton what if we have a split contribution? do they just get 2 separate receipts?

Reply:
@seamuslee001 yeah - I've confirmed that they do so I don't see the point  in 2 trxn receipts